src/components/App/App.tsx

```diff
@@ -1383,44 +1416,26 @@ export default class App extends React.Component<null, AppState> {
                       ></Progress>
                     </div>
                   )}
-                  {this.state.state === 'connected' && (
-                    <VideoChat
-                      socket={this.socket}
-                      participants={this.state.participants}
-                      nameMap={this.state.nameMap}
-                      pictureMap={this.state.pictureMap}
-                      tsMap={this.state.tsMap}
-                      rosterUpdateTS={this.state.rosterUpdateTS}
-                    />
-                  )}
+                  <Footer>
```

Comment:
I would hold off on this change to VideoChat, it might be moved soon to a tab with the Chat.